src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    public Circle(String name, int side, String color) {
+        super(name, side, color);
+    }
+
+    @Override
+    public void getsSquare() {
+        System.out.println("Figure: " + getName()
```

Comment:
Instead of name property you may use class name and predefined method to get it
```suggestion
        System.out.println("Figure: " + getClass().getSimpleName()
```